constants/tasks.js

```diff
@@ -5,11 +5,34 @@ const TASK_TYPE = {
 };
 
 const TASK_STATUS = {
-  ACTIVE: "active",
-  ASSIGNED: "assigned",
-  BLOCKED: "blocked",
-  COMPLETED: "completed",
-  UNASSIGNED: "unAssigned",
+  // old task workflow
+  ACTIVE: "IN_PROGRESS",
+  "HIGH PRIORITY": "HIGH PRIORITY",
+  PENDING: "IN_PROGRESS",
+  ASSIGNED: "ASSIGNED",
+  BLOCKED: "BLOCKED",
+  COMPLETED: "COMPLETED",
+
+  // new task workflow
+  IN_PROGRESS: "IN_PROGRESS",
+  UNASSIGNED: "AVAILABLE",
+  AVAILABLE: "AVAILABLE",
```

Comment:
Two keys with same value?

Reply:
Yes, `unassigned` is from old task values, and `available` is from new status values.